### PR TITLE
docs(flaky-management): add Slack handle troubleshooting section

### PR DESCRIPTION
**Pull request by TovRudyy**
## Summary

Adding a **Troubleshooting** section to the Flaky Tests Management page.

Documents the `@slack-ACCOUNT-CHANNEL` vs `@slack-CHANNEL` handle format edge case: a channel-only handle resolves to the first configured Slack account, which is ambiguous for orgs with multiple Slack workspaces connected.

Do not merge immediately.

### CODE REVIEW

**Review by anmarchenko — APPROVED** [review]

File: content/en/tests/flaky_management/_index.md [hunk 1]
```diff
@@ -236,6 +236,23 @@ To use Flaky Tests Management features, you must use Datadog's native instrument
...
+
+- `@slack-ACCOUNT-CHANNEL` — recommended; routes to the named channel in the specified
+  Slack account (the name you gave the workspace in the [Datadog Slack integration][5]).
+- `@slack-CHANNEL` — works only if your organization has a single Slack account
+  connected. For organizations with multiple Slack workspaces, omitting the account name
+  causes the notification to be routed to the first configured account, which may not be
+  the intended workspace.
+
+To find your account name, go to the [Slack integration tile][5] and check the
+**Account Name** field for the workspace you want to target.
```
Comment (domalessi) [inline content/en/tests/flaky_management/_index.md]:
```suggestion
If Slack notifications are not being delivered, check that your notification rule uses the `@slack-ACCOUNT-CHANNEL` format.

If you are using `@slack-CHANNEL` (without the account name), the notification is routed to the first configured Slack account. For organizations with multiple Slack workspaces, this may not be the intended workspace.

To find your account name, go to the [Slack integration tile][5] and check the
**Account Name** field for the workspace you want to use.
```

**Review by domalessi — COMMENTED** [review]
Thanks for submitting! Left a suggestion for a slight restructure. Ping me when ready for a final look!

**Review by domalessi — APPROVED** [review]
🚀
